src/Aop/Framework/DynamicClosureMethodInvocation.php

```diff
@@ -38,9 +38,7 @@
     public function proceed()
     {
         if (isset($this->advices[$this->current])) {
-            $currentInterceptor = $this->advices[$this->current++];
-
-            return $currentInterceptor->invoke($this);
+            return ($this->advices[$this->current++])->invoke($this);
```

Comment:
If you ask me, I would prefer more that code has 3 lines, like:

    $this->current++;
    $currentInterceptor = $this->advices[$this->current];		
    return $currentInterceptor->invoke($this);

instead of one liner.